contents/docs/session-replay/mobile.mdx

```diff
@@ -11,134 +11,107 @@ Mobile Session Recording allows you to record user sessions on mobile devices. T
 
 ## How it works
 
-We have taken our time to make sure that we can provide a useful and detailed recording experience whilst keeping the performance and security of your app as a top priority. By default the configuration is somewhat resritive with automatic masking and wifi-only uploads to ensure that we don't negatively impact the user experience. This can then be configured to be more or less restrictive depending on your needs.
+We have taken our time to make sure that we can provide a useful and detailed recording experience whilst keeping the performance and security of your app as a top priority. By default the configuration is resritive with automatic masking.
 
 #### Screen recording
 
-Mobile Recording is primarily done using native APIs to grab an image copy of the primary View in the app. This is done carefully so as not to affect performance in any way a user may notice and at a rate of 1 frame per second, so as not to generate too much data. The image is compressed and uploaded to PostHog, by default only over wifi to ensure we aren't negatively impacting the users data plan. 
+Mobile Recording is primarily done using native APIs to grab the view hierarchy state when the screen is drawn. This is done carefully so as not to affect performance in any way a user may notice.
 
-If a `redactionMode` is specified, the image is then processed to remove sensitive information. The different modes work as follows:
-* `selective` (default) - Only redact views with an accessibilityId matching what is specified in `redactionTags`.
-* `automatic` - Redact all views with the name matching `redactionViews` and all classes matching `redactionClasses`. Some sensible defaults are provided to capture anything Text like.
-* `wireframe` - Renders a "wireframe" representation of the views instead of an captured image. This is great for highly sensitive environments where you still wan't to get a sense of user behaviour but don't want to capture any sensitive information.
+The view hierarchy is transformed to a JSON data structure and later rendered as a HTML [wireframe](https://www.figma.com/resource-library/what-is-wireframing/), since it is a wireframe the UI won't have the original look and feel but it should be close enough to understand the user's behaviour.
```

Comment:
```suggestion
The view hierarchy is transformed to a JSON data structure and later rendered as a HTML [wireframe](https://www.figma.com/resource-library/what-is-wireframing/). Since it is a wireframe, the UI won't have the original look and feel but it should be close enough to understand the user's behaviour.
```